bowwowcare/src/views/HomePage/HomePage.js

```diff
@@ -55,16 +57,32 @@ function HomePage() {
         <p className=" text-xs">간단한 진단을 통해 솔루션을 확인해보세요</p>
         </div>
 
-      <div className="flex justify-center mt-6 text-gray-500 text-center text-sm">
-        <button onClick={() => handleNavigation("aggression")} className=" border-r">
-          <img src={aggression}></img>
-          <p>공격적인 아이</p>
-        </button>
+      <div>
+      {isDesktopOrMobile !== true ? 
+        <div className="flex justify-center mt-6 text-gray-500 text-center text-sm">
+          <button onClick={() => handleNavigation("aggression")} className=" border-r">
+            <img src={aggression} style={{width:"250px"}}></img>
```

Comment:
```suggestion
            <img src={aggression} width={isDesktopOrMobile ? "250px" : "auto"} />
```
이런식으로 하는게 더 좋을 것 같은데 이렇게 하면 정상 작동을 안 하나요 ?_?

Reply:
위의 답변을 봐주세용